File: Ejercicio 20/ej20.js
let word1 = "Lapicero";
let word2 = "Copiarle";

let word3 = "Fer";
let word4 = "Asd";

function limpiar(p){
    return p.toLowerCase();
}

function transformToList(p){
    return p.split('').sort().join('');
}

function areAnagrams(p1, p2){
    if(transformToList(limpiar(p1)) == transformToList(limpiar(p2))){
        console.log("son anagramas");
    }else{
        console.log("no son anagramas");
    }
}

areAnagrams(word1, word2);
areAnagrams(word3, word4);

File: Ejercicio 20/ej20.py
word1 = "Lapicero"
word2 = "Copiarle"

word3 = "Fer"
word4 = "Asd"

def limpiar(p: str):
    return p.lower()

def transform_to_list(p: str):
    lista = sorted(list(p))
    return lista

def are_anagrams(p1, p2):
    if transform_to_list(limpiar(p1)) == transform_to_list(limpiar(p2)):
        print("son anagramas")
    else:
        print("no son anagramas")

are_anagrams(word1, word2)
are_anagrams(word3, word4)


